test/acceptance/koa-basic-test.js

```diff
@@ -0,0 +1,61 @@
+const test = require('blue-tape');
+const nf = require('node-fetch');
+
+test('Basic Healthy Koa Health Check', async t => {
+
+  // avoid problems if this env var is already set from wherever test was run
+  process.env.NODE_HEALTH_ENDPOINT_PATH = "";
+
+  const server = getServer();
+  const util = require('../support/util');
+  const baseuri = util.serverUri(server);
+
+  try {
+    const res = await nf(`${baseuri}/hello`);
```

Comment:
What's the reason for this call? It doesn't seem to be playing any role in this test as it stands, and it appears to be 404ing.

Reply:
None at all. It's a leftover copy/paste. Removing. Thanks